packages/xsnap/src/build.js

```diff
@@ -1,70 +1,232 @@
+#!/usr/bin/env node
+/* eslint-disable no-await-in-loop */
```

Comment:
I like to apply this on a case-by-case basis since sometimes it bullies me into using Promise.all when appropriate (parallel). The case in question is definitely serial.

Reply:
indeed, this turned up a job for `Promise.all`